content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -123,6 +124,15 @@ The output shows that `testfile` has group ID 2000, which is the value of `fsGro
 -rw-r--r-- 1 1000 2000 6 Jun  6 20:08 testfile
 ```
 
+Run the following command:
+
+```shell
+$ id
+uid=1000 gid=3000 groups=2000
+```
+You will see that gid is 3000 which is same as `runAsGroup` field. If the `runAsGroup` was ommitted the gid would
+remain as 0(root) and the process would continue to run with some root privileges.
```

Comment:
@kubernetes/sig-node-pr-reviews PTAL at this for accuracy, specifically "the process would continue to run with some root privileges"

Reply:
It doesn't have any root privileges, it just means it's able to interact with root-group files that have group permissions.

Reply:
Doesnt being able to read/write a file owned by 0 group, mean the process has root privileges ?
Note, i have reworded as per suggestion, but i dont agree with the suggestion or missing enough understanding of what root means